webserver/Pages/Account/Login.cshtml

```diff
@@ -34,4 +34,12 @@
             </div>
         </div>
     </form>
+</div>
+
+<div class="form-group row">
+    <div class="col-2">
+        <a href="\Account\Register" class="btn btn-primary">Register</a>
+    </div>
+    <div class="col-5">
+    </div>
 </div>
```

Comment:
We don't want a register buttom.
This is a feature that we should do manually by the admin panel.